static/css/index.css

```diff
@@ -201,6 +201,18 @@ fieldset[disabled] .btn-size.active {
     font-weight: bold;
 }
 
+.pokemon-timer {
+    background-color: gray;
+    border-radius: .25rem !important;
+    color: white;
+    font-weight: bold;
+    font: bold 12px, Monaco, monospace;
+    border: none;
+    border-color: none;
+    box-shadow: none;
+    max-height: 20px;
+}
+
```

Comment:
I think the timer needs more padding to the left and to the right.

![image](https://user-images.githubusercontent.com/3511321/93956738-da366480-fd20-11ea-9ba1-0bf3629ade18.png)